Content.Shared/GameObjects/Components/Nutrition/SharedSmokingComponent.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.GameObjects.Components.Nutrition
+{
+    public class SharedSmokingComponent : Component
+    {
+        public override string Name => "Smoking";
+    }
```

Comment:
abstract, also putting the component in shared when there is no shared functionality is misleading...

If it is gonna be shared one day then also put a componentreference to the shared one on the server-side component and update the code to reference SharedSmokingComponent instead of CigSmokingComponent